vector/src/main/res/layout/motion_dms_fab_menu_merge.xml

```diff
@@ -0,0 +1,79 @@
+    android:layout_height="match_parent"
+    app:layoutDescription="@xml/motion_scene_dms_fab_menu"
+    tools:motionProgress="0.65"
+    tools:parentTag="androidx.constraintlayout.motion.widget.MotionLayout"
+    tools:showPaths="true">
+
+    <View
+        android:id="@+id/dmsCreateRoomTouchGuard"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="?riotx_touch_guard_bg"
+        android:clickable="true"
+        android:contentDescription="@string/a11y_create_menu_close"
+        android:focusable="true" />
+
+    <!-- Sub menu item 2 -->
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
+        android:id="@+id/createDmByQrCode"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="bottom|end"
+        android:accessibilityTraversalBefore="@+id/roomListView"
+        android:contentDescription="@string/a11y_create_direct_message_by_qr_code"
+        android:src="@drawable/ic_fab_add_by_qr_code"
+        app:backgroundTint="#FFFFFF"
+        app:fabCustomSize="48dp"
+        app:maxImageSize="26dp"
+        app:tint="@color/black" />
+
+    <TextView
+        android:id="@+id/createDmByQrCodeLabel"
+        style="@style/VectorLabel"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="8dp"
+        android:ellipsize="end"
+        android:importantForAccessibility="no"
+        android:text="@string/add_by_qr_code" />
+
+    <!-- Sub menu item 1 -->
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
+        android:id="@+id/createDmByMxid"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="bottom|end"
+        android:accessibilityTraversalBefore="@+id/createDmByQrCode"
+        android:contentDescription="@string/a11y_create_direct_message_by_mxid"
+        android:src="@drawable/ic_fab_add_by_mxid"
+        app:backgroundTint="#FFFFFF"
+        app:fabCustomSize="48dp"
+        app:maxImageSize="29dp"
+        app:tint="@color/black" />
+
+    <TextView
+        android:id="@+id/createDmByMxidLabel"
+        style="@style/VectorLabel"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="8dp"
+        android:ellipsize="end"
+        android:importantForAccessibility="no"
+        android:text="@string/add_by_matrix_id" />
+
+    <!-- Menu -->
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
+        android:id="@+id/dmsCreateRoomButton"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:accessibilityTraversalBefore="@+id/createDmByMxid"
+        android:contentDescription="@string/a11y_create_menu_open"
+        android:src="@drawable/ic_fab_add"
```

Comment:
This may need to be replaced, depending on whether a plus sign on this FAB is desired. It looks makes the add group button look a little odd though.